src/main/java/practice/StreamPractice.java

```diff
@@ -35,7 +50,10 @@ public Double getOddNumsAverage(List<Integer> numbers) {
      * Example: select men who can be recruited to army (from 18 to 27 years old inclusively).
      */
     public List<People> selectMenByAge(List<People> peopleList, int fromAge, int toAge) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(p -> p.getSex().equals(People.Sex.MAN)
```

Comment:
better use `==` for constants from enum
```suggestion
                .filter(p -> p.getSex() == People.Sex.MAN
```